bika/lims/browser/worksheet/ajax.py

```diff
@@ -122,6 +122,7 @@ def __call__(self):
         if not mtool.getMemberById(value):
             return
         self.context.setAnalyst(value)
+        self.context.reindexObject()
```

Comment:
Wouldn't be better to add this inside the `setAnalyst` function from `Worksheet`? 
https://github.com/senaite/senaite.core/blob/master/bika/lims/content/worksheet.py#L1695-L1698